README.md

```diff
@@ -232,7 +274,7 @@ You should now have the DB schemas on your localDb:
 Open a terminal, navigate to the Task ```BoosterConf.Ef.Ninja\BoosterConf.Ef.Ninja.TaskD``` folder (it must be the folder where the .csproj file is located). Run the following command:
 
 ```bash
-dotnet ef dbcontext scaffold "Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=EfNinja-TaskD" Microsoft.EntityFrameworkCore.SqlServer
+dotnet ef dbcontext scaffold "Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=BoosterConfEfNinjaTaskOne-TaskD" Microsoft.EntityFrameworkCore.SqlServer
```

Comment:
I suggest we simplify the database names: 
- the DB throughout the task is called "EfNinja"
- the DB provisioned through the script at the end is called "EfNinja-TaskD"